kura/distrib/pom.xml

```diff
@@ -818,6 +818,17 @@
                                     <filemode>600</filemode>
                                 </mapper>
                             </data>
+                            <data>
+                                <src>${project.build.directory}/pkg/packages/</src>
+                                <type>directory</type>
+                                <mapper>
+                                    <type>perm</type>
+                                    <prefix>${kura.install.dir}/${kura.symlink}/packages</prefix>
+                                    <user>root</user>
+                                    <group>root</group>
+                                    <filemode>600</filemode>
+                                </mapper>
+                            </data>
```

Comment:
Given that the packages path is configurable via `kura.properties` is this correct? We would get an unused directory on the filesystem...